packages/mock-account-provider/app/routes/webhooks.ts

```diff
@@ -1,7 +1,8 @@
 import type { ActionArgs } from '@remix-run/node'
 import { json } from '@remix-run/node'
 
-export function action({ request }: ActionArgs) {
-  console.log('received webhook: ', JSON.stringify(request.body))
-  return json(request.body, { status: 201 })
+export async function action({ request }: ActionArgs) {
+  const payload = await request.json()
+  console.log('received webhook: ', JSON.stringify(payload))
+  return json(payload, { status: 201 })
```

Comment:
https://github.com/interledger/rafiki/blob/main/docs/transaction-api.md#webhooks
```suggestion
  return json({ status: 200 })
```